src/components/action-bar/action-bar.scss

```diff
@@ -1,10 +1,18 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-action-bar-expanded-max-width: optionally specify the expanded max width of the action bar when in "vertical" layout.
+ */
+
 :host {
   @extend %component-host;
   @apply pointer-events-auto
     inline-flex
     flex-col
     self-stretch;
-  max-inline-size: 15vw;
+  --calcite-action-bar-expanded-max-width: auto;
```

Comment:
Looking a this name in particular has me thinking if we need them namespaced if they are scoped to the component? If so, we should look into how these are baked into the component styles and whether we need a pattern/util to use a more concise name internally to potentially shave some bits. cc @macandcheese